docs/src/guide/installation.md

```diff
@@ -81,6 +81,9 @@ For a less intrusive way to run all scons scripts with Python 3 add this to your
 
 	alias scons="/usr/bin/env python3 $(which scons)"
 
+It may be that the linux' version of openocd is out of date. In this case install it
+directly from the [openocd][openocd] home page.
+
```

Comment:
```suggestion
The latest openocd release _0.10.0_ (as of May 2020) is too old for some targets (e.g. _STM32G0_, _STM32G4_).
In this case it might be necessary to compile openocd from git _HEAD_.
```